### ISSUE TITLE
lack of details about implementation and acc and asr problem

### ISSUE THREAD

**Issue by andyclsr** [fmy266/Pytorch-Backdoor-Unlearning#1]
Hi, your work is really excellent and I am of great interest to it.
But your provided code is probably not enough to reproduce the result whose loss function and many setting is limited and I tried to implement it but finally get a bad result of it. could you give some more details or notes for the implements? thank you a lot anyway!
